src/plugin.py

```diff
@@ -346,7 +351,12 @@ def create_total_games_cache(self):
         friends_list = current_page['rockstarAccountList']['rockstarAccounts']
         return_list = []
         for i in range(0, len(friends_list)):
-            friend = FriendInfo(friends_list[i]['rockstarId'], friends_list[i]['displayName'])
+            avatar_uri = f"https://a.rsg.sc//n/{friends_list[i]['displayName'].lower()}/l"
+            profile_uri = f"https://socialclub.rockstargames.com/member/{friends_list[i]['displayName']}/"
+            friend = UserInfo(friends_list[i]['rockstarId'],
+                              friends_list[i]['displayName'],
+                              avatar_url=avatar_uri,
+                              profile_url=profile_uri)
```

Comment:
Repeated code - I would use separate method

Reply:
I have added a separate method that uses a list generated by a request to create a new ```List[UserInfo]```.